lib/org/chromium/webidl/PipelineRunner.js

```diff
@@ -19,7 +19,7 @@ foam.CLASS({
     function fmtErrorMsg(msg) {
       return new Error(`${this.cls_.name}: ${msg}`);
     },
-    function run(message) {
+    function validateMessage(message) {
```

Comment:
Discussed offline -> Throwing at the moment will send the error to the RPC message instead. This is not what we want. Until that is fixed, calling `this.error` is appropriate.